itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestTableLevelReplicationScenarios.java

```diff
@@ -313,8 +313,8 @@ public void testIncorrectTablePolicyInReplDump() throws Throwable {
     }
     Assert.assertTrue(failed);
 
-    // Invalid pattern where we didn't enclose table pattern within single or double quotes.
-    replPolicy = primaryDbName + ".[t1].[t2]";
+    // Invalid pattern where where table list is empty.
+    replPolicy = primaryDbName + ".''.'t2'";
```

Comment:
We can treat empty list for dumping only functions. It was decided sometime earlier.

Reply:
it's confusing ..if empty list ..that means include all ? if yes, then user has to give empty exclude list or exclude list as *  for dump only function ? i think for simplicity we should not allow this special case ..if user wants then they can use regex to exclude all tables

Reply:
Yes, agreed.